src/lwc/geFormRenderer/elevateTokenizedGift.js

```diff
@@ -0,0 +1,35 @@
+class ElevateTokenizedGift {
```

Comment:
Because of how this is being used, the name might be better as Elevate**Tokenizeable**Gift.

Reply:
Thanks for the recommendation. I'll go ahead and get that updated.